src/routes/docs/quick-starts/oauth-google/+page.markdoc

```diff
@@ -0,0 +1,353 @@
+
+const account = new Account(client);
+
+client
+  .setEndpoint("https://cloud.appwrite.io/v1") // Your API Endpoint
+  .setProject("[PROJECT_ID]");                 // Your project ID
+
+// Go to OAuth provider login page
+account.createOAuth2Session("google", "[LINK_ON_SUCCESS]", "[LINK_ON_FAILURE]");
+```
+{% /tabsitem %}
+
+{% tabsitem #flutter title="Flutter" %}
+You can use OAuth in your Flutter application, but some platforms like Android and Apple requires additional configuration to enable the OAuth callback, so the your users can be redirected back to your app.
+
+#### Android OAuth callback
+
+In order to capture the Appwrite OAuth callback url, the following activity needs to be added inside the `<application>` tag, along side the existing `<activity>` tags in your `AndroidManifest.xml`. Be sure to replace the `[PROJECT_ID]` string with your actual Appwrite project ID. You can find your Appwrite project ID in the project settings screen in your Appwrite console.
+
+```xml
+<manifest ...>
+  ...
+  <application ...>
+    ...
+    <!-- Add this inside the `<application>` tag, along side the existing `<activity>` tags -->
+    <activity android:name="io.appwrite.views.CallbackActivity" android:exported="true">
+      <intent-filter android:label="android_web_auth">
+        <action android:name="android.intent.action.VIEW" />
+        <category android:name="android.intent.category.DEFAULT" />
+        <category android:name="android.intent.category.BROWSABLE" />
+        <data android:scheme="appwrite-callback-[PROJECT_ID]" />
+      </intent-filter>
+    </activity>
+  </application>
+</manifest>
+```
+
+#### Apple OAuth callback
+
+In order to capture the Appwrite OAuth callback url, the following URL scheme needs to added to your `Info.plist` .Be sure to replace the `[PROJECT_ID]` string with your actual Appwrite project ID. You can find your Appwrite project ID in your project settings screen in your Appwrite console.
+
+```xml
+<key>CFBundleURLTypes</key>
+<array>
+<dict>
+    <key>CFBundleTypeRole</key>
+    <string>Editor</string>
+    <key>CFBundleURLName</key>
+    <string>io.appwrite</string>
+    <key>CFBundleURLSchemes</key>
+    <array>
+        <string>appwrite-callback-[PROJECT_ID]</string>
+    </array>
+</dict>
+</array>
+```
+
+To authenticate a user in your Flutter application, use the [Create OAuth2 Session](https://appwrite.io/docs/client/account?sdk=flutter-default#accountCreateOAuth2Session) endpoint.
+
+```dart
+import 'package:appwrite/appwrite.dart';
+void main() async {
+    final client = new Client();
+    final account = new Account(client);
+    client
+        .setEndpoint('https://cloud.appwrite.io/v1') // Your API Endpoint
+        .setProject('[PROJECT_ID]')                  // Your project ID
+    ;
+    // OAuth Login, for simplest implementation you can leave both success and
+    // failure link empty so that Appwrite handles everything.
+    await account.createOAuth2Session(provider: 'google');
```

Comment:
This is true for mobile, but not for flutter web. For flutter web, the success and failure should link to the auth.html page that was configured.